pages/linux/perf.md

```diff
@@ -0,0 +1,19 @@
+# perf
+
+> Framework for linux performance counter measurements.
+
+- Gather basic performance counter stats for a command:
+
+`perf stat {{command}}`
```

Comment:
I think it might be clearer to mention an actual command so that users can get a better idea. Also add a `-` after stat so that extra options can be applied to a command.